model/model.go

```diff
@@ -0,0 +1,128 @@
+package model
+
+import (
+	"database/sql/driver"
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+// ValidatorInfo is a model for the validator_info table.
+// query stakeHub getValidators at startup only once and only their validator?
+type ValidatorInfo struct {
+	ID uint64 `gorm:"column:id;primaryKey"`
+
+	Operator  common.Address `gorm:"column:operator;uniqueIndex:idx_operator"`
+	Consensus common.Address `gorm:"column:consensus"`
+	Credit    common.Address `gorm:"column:credit"`
+}
+
+// Validator is a model for the validator table.
+type Validator struct {
+	ID uint64 `gorm:"column:id;primaryKey"`
+
+	Operator common.Address `gorm:"column:operator;uniqueIndex:idx_operator_date"`
```

Comment:
This should not be uniqueIndex, because we will have duplicated record for one validator